### PR TITLE
Rename environment package to vars + minor clean up

### PR DESCRIPTION
**Pull request by pdalpra**

### CODE REVIEW

File: lib/vars/vars.go [hunk 1]
```diff
@@ -28,26 +29,23 @@ func DockerNameFriendly(baseName string) string {
...
-		return os.Getwd()
+		wd, err := os.Getwd()
+		if err != nil {
+			return "", nil
+		}
+		return filepath.Base(wd), nil
 	})
 	if err != nil {
-		return "", err
+		return "", nil
```
Comment (gbougeard) [inline lib/vars/vars.go]: swallow the error?

Comment (pdalpra) [inline lib/vars/vars.go]: woops, not intended :D

Comment (pdalpra) [inline lib/vars/vars.go]: fixed

**Comment by gbougeard** [comment]
LGTM

**Pull request merged**